src/ressa/node_pattern.rs

```diff
@@ -69,6 +104,16 @@ impl NodePattern {
     }
 }
 
+impl NodeLanguage for NodePattern {
+    fn get_language(&self) -> Language {
+        if let Some(language) = self.language {
+            language.clone()
+        } else {
+            Language::default()
+        }
+    }
```

Comment:
A nice shorthand for this is
```rust
self.language.unwrap_or_default()
```

Reply:
Interesting. Last time I tried using this, it told me it was a nightly-only feature. Just tried it again, and it works like a charm.